### ISSUE TITLE
Duplicate error occurs when use v7-kotlin, recyclerview-v7-kotlin and mulitDex.

### ISSUE THREAD

**Issue by satorufujiwara** [JakeWharton/RxBinding#143]
Hello, I use `rxbinding-appcompat-v7-kotlin` and `rxbinding-recyclerview-v7-kotlin`.

ZipException occurs when use mulitDex.
It occurs because they have same package `com.jakewharton.rxbinding.support.v7.widget`.

```
Error:Execution failed for task ':mobile:packageAllDevDebugClassesForMultiDex'.
> java.util.zip.ZipException: duplicate entry: com/jakewharton/rxbinding/support/v7/widget/WidgetPackage.class      
 ```
 
 build.gradle is here.
 ```         
defaultConfig {
    multiDexEnabled true
}
dependencies {
    compile "com.jakewharton.rxbinding:rxbinding-appcompat-v7-kotlin:0.2.0"
    compile "com.jakewharton.rxbinding:rxbinding-recyclerview-v7-kotlin:0.2.0"
}
```

**Comment by JakeWharton** [comment]
This will be fixed when we update to M13.

**Comment by satorufujiwara** [comment]
Thanks! I look forward to it.

**Comment by JakeWharton** [comment]
Can you try the latest SNAPSHOT to verify?

**Comment by satorufujiwara** [comment]
Thank you for your update.
I tried it but same error occurred.
RxBinding's version is below.
```
0.3.0-SNAPSHOT/rxbinding-appcompat-v7-kotlin-0.3.0-20150928.062152-6.aar
0.3.0-SNAPSHOT/rxbinding-recyclerview-v7-kotlin-0.3.0-20150928.062209-6.aar
```

I tried with this project.
https://github.com/satorufujiwara/kotlin-android-example/tree/kotterknife

**Comment by JakeWharton** [comment]
Ok I'll take a look!

**Comment by JakeWharton** [comment]
This is fixed post-M14 as they changed the name of the generated classes (and should be fixed on `master` for the next release). Here's the output of a clean build:
<img width="323" alt="screen shot 2015-10-22 at 6 32 11 pm" src="https://cloud.githubusercontent.com/assets/66577/10680290/51cd2754-78ec-11e5-87f5-0dd690aa7ee7.png">
<img width="377" alt="screen shot 2015-10-22 at 6 38 37 pm" src="https://cloud.githubusercontent.com/assets/66577/10680294/579f407c-78ec-11e5-86f5-344909042f00.png">

**Comment by satorufujiwara** [comment]
I confirmed that this issue is fixed with rxbinding 0.3.0 and kotlin 1.0.0-beta.
Thank you!
